jest.unit.conf.json

```diff
@@ -40,18 +40,18 @@
   "coverageReporters": ["text", "html", "lcov"],
   "collectCoverageFrom": [
     "frontend/src/**/*.{js,jsx,ts,tsx}",
-    "enterprise/frontend/src/**/*.{js,jsx,ts,tsx}"
+    "enterprise/frontend/src/**/*.{js,jsx,ts,tsx}",
+    "!<rootDir>/**/*.styled.{js,jsx,ts,tsx}",
+    "!<rootDir>/**/*.story.{js,jsx,ts,tsx}",
+    "!<rootDir>/**/*.info.{js,jsx,ts,tsx}",
+    "!<rootDir>/**/*.unit.spec.{js,jsx,ts,tsx}"
   ],
   "coveragePathIgnorePatterns": [
     "/node_modules/",
     "/frontend/src/metabase/visualizations/lib/errors.js",
     "/frontend/src/cljs/",
     "/frontend/src/metabase/internal/",
-    "/frontend/test/",
-    ".styled.{jsx,tsx}",
-    ".info.js",
-    "**/*.unit.*",
-    "**/*.stories.{js,jsx,ts,tsx}"
+    "/frontend/test/"
   ],
```

Comment:
I remember running into the same problem a long time ago — somehow, I couldn't find a way to exclude files by a regexp in `coveragePathIgnorePatterns`, but copy-pasting the same pattern to `collectCoverageFrom` with a `!` at the beginning makes it magically work 🤷